tools/metaquantome/metaquantome_viz.xml

```diff
@@ -389,15 +387,25 @@
                 </assert_contents>
             </output>
         </test>
-        <test expect_num_outputs="3">
-             <param name="infile" value="tax_test_out.tab" ftype="tabular"/>
-             <param name="mode" value="t" />
-             <param name="samps" value="rudney_samples.tab" ftype="tabular"/>
-             <param name="plottype" value="heatmap" />
-             <output name="wrapped_outfile">
-                <assert_contents>
-                    <has_text text="To download the image" />
-                </assert_contents>
+        <test>
```

Comment:
can you please add `expect_num_outputs` to the tests. See here: https://docs.galaxyproject.org/en/latest/dev/schema.html#id80